### ISSUE TITLE
Create the class line in base component 

### ISSUE THREAD

**Issue by fabiorogeriosj** [MobileUI/components#92]
The `.line` just add line horizontal

**Comment by daniloff200** [comment]
hi :)
so, I just need to add a `.line` class, which should draw a horizontal line? 
For example, like that

```
.line {
    display: block;
    height: 1px;
    border: 0;
    border-top: 1px solid #ccc;
    margin: 1em 0;
    padding: 0;
}
```
is it okay?

**Comment by fabiorogeriosj** [comment]
that's right, but I suggest you use the `background` to paint the background, so we can use the color classes, for example:

```
<div class="line purple"></div>
```

in this example I use class `line` and `purple`, the result is line purple.

**Comment by galloj** [comment]
@fabiorogeriosj This issue can be closed #93
